app/assets/stylesheets/autotune/_core.scss

```diff
@@ -0,0 +1,888 @@
+//*\\//*\\//*\\//*\\//*\\
+// Main
+//*\\//*\\//*\\//*\\//*\\
+
+html{
+  overflow-x:hidden;
+  height: auto;
+}
+
+h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6{
```

Comment:
This section on blueprints and themes.
![screen shot 2016-07-06 at 12 11 42 pm](https://cloud.githubusercontent.com/assets/3526984/16625487/d532b8ce-4372-11e6-94a6-b7776721e037.png)